### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by ianliu** [zeapo/Android-Password-Store#2702] [Q&A] [answered]
The latest update on F-droid / Google Play is from 2021, but I see very recent commits. Do you guys compile from source to use it?

**Comment by msfjarvis** [comment, marked as answer]
The project publishes officially supported snapshot builds of the latest code, so compiling it yourself isn't necessary. See https://docs.passwordstore.app/docs/users/release-channels for more info.

Regarding being "dead", 🤷. I'm a lone maintainer and don't have much incentive to do a whole lot more than where the app is right now. The gap between what's good enough for my needs and what's necessary for the app to carry a "stable" moniker is sizable, and closing that gap has been a slow process.
